docs/README.md

```diff
@@ -538,7 +538,6 @@ Here you can find all the supported libraries you can use in you Kotlin notebook
  - [biokotlin](https://github.com/maize-genetics/BioKotlin) - BioKotlin aims to be a high-performance bioinformatics library that brings the power and speed of compiled programming languages to scripting and big data environments.
  - [combinatoricskt](https://github.com/shiguruikai/combinatoricskt) - A combinatorics library for Kotlin
  - [coroutines](https://github.com/Kotlin/kotlinx.coroutines) - Asynchronous programming and reactive streams support
- - [database](https://github.com/Kotlin/kotlin-jupyter-database) - Database Connection Helper Functions
```

Comment:
Hello @cmelchior . Is https://github.com/Kotlin/kotlin-jupyter-database a private repository? I've seen links to this repo in places other than here as well.

Reply:
It is currently private, but that is unintentional, so I just prepared the documentation for when it is moved to GitHub. This will probably happen next week.